modules/phase_field/include/utils/EBSDAccessFunctors.h

```diff
@@ -21,8 +21,8 @@ class EBSDAccessFunctors
 public:
   /// Per element EBSD data point
   struct EBSDPointData {
-    Real phi1, phi, phi2, symmetry;
-    unsigned int grain, phase, op, global;
+    Real phi1, Phi, phi2;
```

Comment:
The second Phi is a capital Phi. If you Google EBSD Euler angles you'll
find plenty of references. HTH, HAND

On Thu, Feb 25, 2016, 8:08 AM Cody Permann <notifications@github.com> wrote:

> In modules/phase_field/include/utils/EBSDAccessFunctors.h
> <https://github.com/idaholab/moose/pull/6405#discussion_r54108380>:
>
> > @@ -21,8 +21,8 @@ class EBSDAccessFunctors
> >  public:
> >    /// Per element EBSD data point
> >    struct EBSDPointData {
> > -    Real phi1, phi, phi2, symmetry;
> > -    unsigned int grain, phase, op, global;
> > +    Real phi1, Phi, phi2;
>
> No hostility, you already convinced me that this is "the normal way". I
> just wanted to know if there was a reference or some wiki page or
> something. If not, fine, just asking.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/idaholab/moose/pull/6405/files#r54108380>.
>